<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointWin</class>
 <widget class="QDialog" name="AppointWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>任命版主</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>请选择版块</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>恋爱交友</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>升学交流</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学习交流</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>运动健康</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>实习工作</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>休闲娱乐</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>星座占卜</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>外出旅游</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>请选择用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>3000</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>普通用户</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_general">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>341</width>
             <height>291</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_10">
          <attribute name="title">
           <string>版主</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_moderator">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>351</width>
             <height>261</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="appoint_btn">
       <property name="text">
        <string>任命</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>卸任版主</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>请选择版块</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>请选择用户</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>330</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>330</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>6</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>恋爱交友</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>5000</width>
             <height>5000</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>升学交流</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>5000</width>
             <height>5000</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>学习交流</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>5000</width>
             <height>5000</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>运动健康</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_4">
           <property name="geometry">
            <rect>
             <x>-2</x>
             <y>-2</y>
             <width>5001</width>
             <height>5001</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>实习工作</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_5">
           <property name="geometry">
            <rect>
             <x>-2</x>
             <y>-1</y>
             <width>5001</width>
             <height>4991</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>休闲娱乐</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_6">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>-1</y>
             <width>4881</width>
             <height>4931</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>星座占卜</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_7">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>0</y>
             <width>5001</width>
             <height>5001</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>外出旅游</string>
          </attribute>
          <widget class="QListWidget" name="listWidget_8">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>5000</width>
             <height>5000</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="disappoint_btn">
       <property name="text">
        <string>卸任</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
